### PR TITLE
openshift/Dockerfiles: apply bundler fix for .so dir

### PR DESCRIPTION
**Pull request by davidor**
The problem still happens in the bundler version included in rh-scl 2.5

### CODE REVIEW

File: openshift/distro/centos/7/Dockerfile [hunk 1]
```diff
@@ -111,12 +111,11 @@ RUN source scl_source enable ${RUBY_SCL} \
...
     fi \
  && bundle install --deployment --jobs $(grep -c processor /proc/cpuinfo) \
-# Bundler < 1.12.0 needs fixing for git gems with extensions (Puma)
- && if ruby -e "begin; require 'rubygems'; Gem::Specification.find_by_name('bundler').version < Gem::Version.new('1.12.0') && exit(0); rescue; end; exit(1)"; then \
-      echo "[WARNING] Old Bundler requires fixing git gems with extensions (Puma)" ; \
-      PUMA_DIR=$(bundle show puma) \
-      && ln -s $(find "${PUMA_DIR}" -name puma_http11.so) "${PUMA_DIR}"/lib/puma ; \
-    fi \
+
+ ## bundler does not place the .so in the correct dir
```
Comment (unleashed) [inline openshift/distro/centos/7/Dockerfile]: just one `#` is enough ;)

**Comment by unleashed** [comment]
which bundler version is this?

**Comment by davidor** [comment]
1.16.1

File: openshift/distro/centos/7/Dockerfile (hunk 1 above)
Comment (unleashed) [reply openshift/distro/centos/7/Dockerfile]: this is rather weird - something from the ruby configuration, rubygems or bundler (or even puma using some wrong directory?) might be changing from a local installation (non-RHSCL) which I can't reproduce

Comment (davidor) [reply openshift/distro/centos/7/Dockerfile]: I can't reproduce it either on a non-rhscl env.

**Comment by unleashed** [comment]
bors flip-flip-all-the-tables

**Comment by unleashed** [comment]
@davidor we should probably dig a bit to find out the cause of this and open an issue somewhere... as time permits.

**Comment by davidor** [comment]
Sure, it was important to merge this because the listener cannot even boot after the ruby upgrade using these images. Let's try to understand why that bundler version is broken later.

**Pull request merged**